Automattic/jetpack
# Photon: Add webp support

Supports #19654

#### Changes proposed in this Pull Request:

* Adds webp support to Photon

#### Jetpack product discussion


n/a


#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Using WP 5.8, upload a webp image to your site (see test images at https://developers.google.com/speed/webp/gallery1 )
* Ensure the image CDN feature is active.
* Before the patch, see that Photon URLs are not used for the webp image.
* After patch, see that they are.
